src/announcement.iced

```diff
@@ -12,7 +12,9 @@ exports.Announcement = class Announcement extends Base
 
   _type : () -> constants.sig_types.announcement
 
-  _json : () -> 
+  _required_stanzas : () -> super.concat(["announcement"])
```

Comment:
I was surprised to learn that `super` is implicitly the same as `super()`. Could we use the more explicit version?